lightning/src/util/events.rs

```diff
@@ -515,6 +535,12 @@ impl Writeable for Event {
 					(2, payment_hash, required),
 				})
 			},
+			&Event::OpenChannelRequest { ref open_channel_request_info } => {
+				11u8.write(writer)?;
```

Comment:
11 is already used for `DiscardFunding`. We need a new value here. That said, we shouldn't actually ever load this object from disk, similar to how we handle `PendingHTLCsForwardable` and `FundingGenerationReady` - if we have such an object when we go to serialize, it won't make sense when we go to reload it. In lightning, if a channel has not exchanged funding messages, it is dropped on reconnect (ie restart), so the peer will have forgotten about the channel entirely anyway.

Reply:
Thanks for the feedback! I'll make sure to fix that.